recipes/r-clues/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = '0.5.9' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-clues
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: clues_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/clues_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/clues/clues_{{ version }}.tar.gz
+  sha256: 1e6a986cb2a90a343aff625774e071b536c8de923373cdc49dbc7c5bddd2d47e
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base{indent}libgcc  # [not win]
```

Comment:
This is the line causing you problems. It should be something like:

```
  - r-base
  - libgcc  # [not win]
```

The `{indent}` error is a known bug in conda-build that has been fixed. I'd recommend updating your version: `conda install -c conda-forge conda-build`. Also, I'd recommend the [conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper) to make it easier to pass the linter checks.

Reply:
@jdblischak Thanks!